deploy/docker/hail_search/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM hailgenetics/hail:0.2.115
```

Comment:
for the protoype, we used a much simpler dockerfile - https://github.com/broadinstitute/seqr/blob/d2c4b3c7022166c66fdd50113eb8e16551551482/deploy/docker/hail_search/Dockerfile

Is there a reason to add all this here? I really liked having a slim image with no additional deployment requirements, and I feel like gunicorn is really overkill here

Reply:
It's all gunicorn, except for the readiness_probe, which probably need.  It's possible to configure the readiness probe entirely in k8s though, rather than having a command, which we can opt for.